### PR TITLE
Consolidate shortcuts and update notebook command handling

### PR DESCRIPTION
**Pull request by blink1073**

### CODE REVIEW

File: src/notebook/plugin.ts [hunk 1]
```diff
@@ -179,6 +194,42 @@ class NotebookFileHandler extends AbstractFileHandler {
     return ['.ipynb']
   }
 
+  executeSelectedCell(): void {
+    for (let w of this.widgets) {
+      if (w.hasClass('jp-mod-focus')) {
```
Comment (jasongrout) [inline src/notebook/plugin.ts:199]: The notebook shouldn't depend on the magic value of the focus class that is defined in the application plugin. I think it's probably okay to not even do the focus check here.

Comment (blink1073) [inline src/notebook/plugin.ts:199]: It is defined in `jupyter-js-docmanager`.  We'd be duplicating the `node.contains` logic there.

Comment (blink1073) [inline src/notebook/plugin.ts:199]: Another option is to move `activeWidget` into `AbstractFileHandler` and let the document manager set it.

**Comment by blink1073** [comment]
Superseded by #72.

**Pull request closed without merging**